### DISCUSSION TITLE
公式标记和引用的全角括号问题

### DISCUSSION THREAD

**Discussion by ZhangSongyi** [obster-y/XJTU-thesis#114] [General]
学校提供的word模板中，对公式注记的规则为

    （2）公式后应注明编号，公式号应置于小括号中，如公式（2-3）。写在右边行末，中间不加虚线；

虽然这里没有明确的说明，但所有例子里都使用了全角括号`（）`而非半角括号`()`
不知道能否把这个规则同步到所有公式环境、`eqref`和`cref`中。

**Comment by fei-wang-1314** [comment]
中文论文，不应该就是全角吗？

**Reply by sikouhjw** [reply]
不应该

**Comment by fei-wang-1314** [comment]
> 不应该

![image](https://github.com/obster-y/XJTU-thesis/assets/77833298/78e19c8b-4e9f-4727-b4a9-c156e5da9a6f)

现在的latex版本是半角
